doc/time-series.rst

```diff
@@ -129,6 +129,8 @@ the first letters of the corresponding months.
 
 You can use these shortcuts with both Datasets and DataArray coordinates.
 
+In addition, xarray supports rounding operations ``floor``, ``ceil``, and ``round``. These operations require you to supply a rounding frequency as a string argument (see list of frequency aliases at http://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases).
```

Comment:
please make a link instead of printing the URL, e.g.,

```
These operations require you to supply a rounding `frequency as a string argument`_.

.. _frequency as a string argument: http://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases
```